src/Silverback.Integration/Messaging/Validation/MessageValidationException.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2020 Sergio Aquilini
+// This code is licensed under MIT license (see LICENSE file for details)
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.Serialization;
+
+namespace Silverback.Messaging.Validation
+{
+    /// <summary>
+    ///     The exception that is thrown when the message is not valid and the validation mode is 'ThrowException' (<see cref="MessageValidationMode" />).
```

Comment:
You can link the `ThrownException` directly via `<see cref="MessageValidationMode.ThrowException" />`.